diff --git a/printer.cfg b/printer.cfg
index 7c6c812..386b23a 100644
--- a/printer.cfg
+++ b/printer.cfg
@@ -21,7 +21,7 @@ endstop_pin : tmc2209_stepper_x:virtual_endstop
 microsteps : 16
 full_steps_per_rotation : 200
 step_distance : 0.012460
-position_endstop : -2.0
+position_endstop : 256.499
 position_max : 256.5
 position_min : -2.01
 homing_accel : 5000
@@ -49,7 +49,7 @@ homing_current : 0.8
 homing_speed : 80.000000
 second_homing_speed : 80.000000
 homing_retract_dist : 0
-homing_force_retract : 20
+homing_force_retract : 30
 virtrual_endstop : 0
 
 [stepper_z]
